### PR TITLE
prevent freeze when using dev server without bail

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -30,13 +30,17 @@ export class UnusedFilesWebpackPlugin {
...
     const absolutePathResolver = it => joinPath(globOptions.cwd, it);
+    const errorArray = this.options.failOnUnused ?
+      compilation.errors :
+      compilation.warnings;
 
     const handleError = err => {
-      if (compilation.bail) {
-        done(err);
-      } else {
-        compilation.errors.push(err);
```
Comment (graingert) [inline src/index.js]: missing call to done() here.

**Comment by graingert** [comment]
@tomchentw ping

**Comment by graingert** [comment]
@tomchentw ping ping ping

**Comment by tomchentw** [comment]
Released [3.0.1](https://github.com/tomchentw/unused-files-webpack-plugin/releases/tag/v3.0.1)

**Pull request merged**